functests/ptp/ptp.go

```diff
@@ -108,4 +96,437 @@ var _ = Describe("ptp", func() {
 			}
 		})
 	})
+	var _ = Describe("PTP e2e tests", func() {
+		var runningPod *v1core.PodList
+		var ptpRunnungPods []v1core.Pod
+		var _ = Context("PTP Interfaces discovery", func() {
+
+			BeforeEach(func() {
+				var err error
+				ptpRunnungPods = nil
+				runningPod = nil
+				runningPod, err = client.Client.Pods(ptpLinuxDaemonNamespace).List(metav1.ListOptions{LabelSelector: "app=linuxptp-daemon"})
```

Comment:
if this is local, you can have `runningPods, err := ...`
Also, I'd rename it to ptpPods or just pods (as the scope is narrow enough)